artmaker/src/main/java/io/artmaker/ArtMakerUIState.kt

```diff
@@ -28,4 +28,7 @@ data class ArtMakerUIState(
     val canRedo: Boolean = false,
     val canUndo: Boolean = false,
     val canClear: Boolean = false,
+    val shouldUseStylusOnly: Boolean = false,
+    val canShowEnableStylusDialog: Boolean = true,
```

Comment:
When working on the changes requested by @CalebKL [here](https://github.com/Fbada006/ArtMaker/pull/72#discussion_r1729807044), I realise that we need to show the stylus dialog at least once, hence the extra setting here. The preferences also have a new key

Reply:
This is noted...